Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_8.yml

```diff
@@ -3553,18 +3565,47 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-    scriptarguments:
-      URL:
-        complex:
-          root: URL
-          accessor: Data
-          transformers:
-          - operator: uniq
-    loop:
+  "228":
+    id: "228"
+    taskid: 2959e8cf-1f8a-4e95-83a6-0dcbd39f64f2
+    type: condition
+    task:
+      id: 2959e8cf-1f8a-4e95-83a6-0dcbd39f64f2
+      version: -1
+      name: Should be reviewed by analyst?
+      type: condition
       iscommand: false
-      exitCondition: ""
-      wait: 1
-      max: 100
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "226"
+      "yes":
+      - "7"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isTrue
```

Comment:
usually, we use an equals transformer so the input will not be changed to 1 for example ( which also can be true).